php/libraries/PageElementsFormatter.class.inc

```diff
@@ -0,0 +1,164 @@
+<?php
+/**
+ * This file contains the Loris Page Element Formatter.
+ * This class is used to format a form element into the
+ * format outline in the LORIS Instrument API documentation.
+ *
+ * PHP Version 7
+ *
+ * @category Main
+ * @package  Behavioural
+ * @author   Jordan Stirling <jstirling91@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris-Trunk/
+ */
+
+/**
+ * This class is used to format a form element into the
+ * format outline in the LORIS Instrument API documentation.
+ *
+ * @category Main
+ * @package  Behavioural
+ * @author   Jordan Stirling <jstirling91@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris-Trunk/
+ */
+class PageElementsFormatter
+{
+
+    /**
+     * Base method to format an form element
+     *
+     * @param array  $element the form element to format
+     * @param string $elName  the element name
+     *
+     * @return array the formatted element
+     */
+    static function formatElement($element, $elName)
+    {
+        $tempElement = array(
+                        "Type"        => $element['type'],
```

Comment:
How do you make sure the key 'type' exists in the array?
Is the function intended to "eat" `LorisFormElement` ?

Reply:
Yeah, this class is designed so that it could hypothetically replace the current way we convert the instrument to JSON for the API. The inputted element is expected to be in the format outputted by LorisForm which as far as I know always has the type set.

Reply:
Could you type in `$element` to `\LorisFormElement`?